source/stylesheets/pages/_guides.scss

```diff
@@ -1,3 +1,18 @@
+.video-wrapper {
+  position: relative;
+  padding-bottom: 56.25%; /* 16:9 */
+  padding-top: 25px;
```

Comment:
✂️ This adds to that `16:9` and makes it not `16:9`.

Let's also move this CSS into it's own component file